packages/mobile/src/transactions/saga.ts

```diff
@@ -115,6 +116,8 @@ export function* sendAndMonitorTransaction<T>(
     if (STABLE_CURRENCIES.some((stableCurrency) => balancesAffected.has(stableCurrency))) {
       yield put(fetchStableBalances())
     }
```

Comment:
Should we remove this?

Reply:
Like I said before, eventually yes, but while we still have the balances in the `stableToken` reducer we need it to be up to date :/